spec/manageiq_performance/middleware_storage/memory_spec.rb

```diff
@@ -0,0 +1,55 @@
+require "manageiq_performance/middleware_storage/memory"
+
+describe ManageIQPerformance::MiddlewareStorage::Memory do
+  let(:env)         { {} }
+  let(:data)        { { :sql_queries => %w[Query1 Query2] } }
+  let(:data_writer) { Proc.new { data } }
+
+  describe "#initialize" do
+    # Make sure the subject is initialized in each spec to reset last_run
+    before(:each) { subject }
```

Comment:
Rebased in this fix for a spec failure that was happening when running with this seed:

```console
$ rspec --seed 19963
```

Effectively, some of the tests farther down were being run, but a new version of the MemoryStorage was not being initialized before running the "reset the ManageIQPeformance.last_run" spec, so it would not have a `nil` value.  So means the code for last run is working... but it wasn't checking after a new MemoryStorage was created, causing a failure.